http://bugs.gentoo.org/165987

2006-10-31  Joseph S. Myers  <joseph@codesourcery.com>

	* sysdeps/arm/eabi/fclrexcpt.c: Include <sysdep.h> instead of
	<asm/procinfo.h>.  Use HWCAP_ARM_VFP instead of HWCAP_VFP.
	* sysdeps/arm/eabi/fedisblxcpt.c: Likewise.
	* sysdeps/arm/eabi/feenablxcpt.c: Likewise.
	* sysdeps/arm/eabi/fegetenv.c: Likewise.
	* sysdeps/arm/eabi/fegetexcept.c: Likewise.
	* sysdeps/arm/eabi/fegetround.c: Likewise.
	* sysdeps/arm/eabi/feholdexcpt.c: Likewise.
	* sysdeps/arm/eabi/fesetenv.c: Likewise.
	* sysdeps/arm/eabi/fesetround.c: Likewise.
	* sysdeps/arm/eabi/fraiseexcpt.c: Likewise.
	* sysdeps/arm/eabi/fsetexcptflg.c: Likewise.
	* sysdeps/arm/eabi/ftestexcept.c: Likewise.
	* sysdeps/arm/eabi/setfpucw.c: Likewise.

===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fclrexcpt.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fclrexcpt.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fclrexcpt.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 __feclearexcept (int excepts)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned long int temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fedisblxcpt.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fedisblxcpt.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fedisblxcpt.c	2006/10/31 17:20:42	1.2
@@ -24,12 +24,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 fedisableexcept (int excepts)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned long int new_exc, old_exc;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/feenablxcpt.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/feenablxcpt.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/feenablxcpt.c	2006/10/31 17:20:42	1.2
@@ -24,12 +24,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 feenableexcept (int excepts)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned long int new_exc, old_exc;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fegetenv.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fegetenv.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fegetenv.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 __fegetenv (fenv_t *envp)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned long int temp;
       _FPU_GETCW (temp);
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fegetexcept.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fegetexcept.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fegetexcept.c	2006/10/31 17:20:42	1.2
@@ -24,12 +24,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 fegetexcept (void)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned long temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fegetround.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fegetround.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fegetround.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 fegetround (void)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned int temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/feholdexcpt.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/feholdexcpt.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/feholdexcpt.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 feholdexcept (fenv_t *envp)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned long int temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fesetenv.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fesetenv.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fesetenv.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 __fesetenv (const fenv_t *envp)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       unsigned int temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fesetround.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fesetround.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fesetround.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 fesetround (int round)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       fpu_control_t temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fraiseexcpt.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fraiseexcpt.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fraiseexcpt.c	2006/10/31 17:20:42	1.2
@@ -24,12 +24,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 feraiseexcept (int excepts)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       int fpscr;
       const float fp_zero = 0.0, fp_one = 1.0, fp_max = FLT_MAX,
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/fsetexcptflg.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/fsetexcptflg.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/fsetexcptflg.c	2006/10/31 17:20:42	1.2
@@ -24,12 +24,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 __fesetexceptflag (const fexcept_t *flagp, int excepts)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       fexcept_t temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/ftestexcept.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/ftestexcept.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/ftestexcept.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 int
 fetestexcept (int excepts)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       fexcept_t temp;
 
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/arm/eabi/setfpucw.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ports/sysdeps/arm/eabi/setfpucw.c	2005/10/10 15:29:32	1.1
+++ ports/sysdeps/arm/eabi/setfpucw.c	2006/10/31 17:20:42	1.2
@@ -23,12 +23,12 @@
 #include <unistd.h>
 #include <ldsodefs.h>
 #include <dl-procinfo.h>
-#include <asm/procinfo.h>
+#include <sysdep.h>
 
 void
 __setfpucw (fpu_control_t set)
 {
-  if (GLRO (dl_hwcap) & HWCAP_VFP)
+  if (GLRO (dl_hwcap) & HWCAP_ARM_VFP)
     {
       fpu_control_t cw;
 
